examples/02_model_collaborative_filtering/multi_vae_deep_dive.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>it is not clear if the code is something you developed or if it is based on other people work. If you develop the code, then it is ok to say "we use, we did, etc", otherwise, it is better to say "the authors use, the authors did..."</p><p> </p><p>Also, please remove all reference to "it is straightforward..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I concur. There are many "we" whereas the text is following closely the papers. It would be better to say "according to" and cite Liang et al. or Higgins et al. as appropriate.

Reply:
The model is developed from scratch based on the examined paper. 

We will make the necessary changes in order to be clear that we refer to the paper.